packages/design-system/src/components/Badge/Badge.test.tsx

```diff
@@ -1,42 +1,50 @@
 import React from 'react';
-import Badge from './Badge';
+import Badge, { BadgeProps } from './Badge';
 import { render, screen } from '@testing-library/react';
 
-describe('Badge', () => {
-  it('should render a default badge', () => {
-    render(<Badge>Foo</Badge>);
-    expect(screen.getByText('Foo')).toMatchSnapshot();
-  });
-
-  it('should render an info badge', () => {
```

Comment:
i removed many snaps for badge because our VRT should catch all the variations of badge. we just need to test that changing the `variation` prop will do its job once.

Reply:
💯